### PR TITLE
Auto OCP-15992

### PR DESCRIPTION
**Pull request by huiran0826**
Logs: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/59622/console

Can you help review? Thanks!
@zhaozhanqi @weliang1  @anuragthehatter @rbbratta

### CODE REVIEW

File: features/networking/egress-ip.feature [hunk 1]
```diff
@@ -178,3 +178,42 @@ Feature: Egress IP related features
...
+  @destructive
+  Scenario: The EgressNetworkPolicy should work well with egressIP
+    Given the valid egress IP is added to the node
+    And I have a project
+    And I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/egressnetworkpolicy/limit_policy.json |
+      | n | <%= project.name %> |
```
Comment (weliang1) [inline features/networking/egress-ip.feature]: indentation

File: features/networking/egress-ip.feature [hunk 2]
```diff
@@ -178,3 +178,42 @@ Feature: Egress IP related features
...
+    Then the step should succeed
+
+    # add the egress ip to the project
+    Given as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
+      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
```
Comment (weliang1) [inline features/networking/egress-ip.feature]: Is cb.valid_ip come from "Given the valid egress IP is added to the node"?

File: features/networking/egress-ip.feature [hunk 3]
```diff
@@ -178,3 +178,42 @@ Feature: Egress IP related features
...
+      | p             | {"spec":{"egress":[{"type":"Allow","to":{"cidrSelector": "0.0.0.0/0"}}]}} |
+      | n             | <%= project.name %>                                                       |
+      | type          | merge                                                                     |
+    Then the step should succeed
+
+    # The traffic should be allowed and the source ip is egress ip
+    When I execute on the pod:
+      | curl | -s | --connect-timeout | 5 | ifconfig.me |
+    Then the step should succeed
+    And the output should contain "<%= cb.valid_ip %>"
```
Comment (weliang1) [inline features/networking/egress-ip.feature]: Should you add last step to clean up policy1 at the end?

Comment (anuragthehatter) [reply features/networking/egress-ip.feature]: @weliang1 Policy1 will be deleted automatically when project gets deleted post scenario execution. @huiran0826 You can 207-213 into 2 lines by using ```Given ad admin i successfuly merge``` step

File: features/networking/egress-ip.feature (hunk 1 above)
Comment (huiran0826) [reply features/networking/egress-ip.feature]: Thanks, will fix it.

File: features/networking/egress-ip.feature (hunk 2 above)
Comment (huiran0826) [reply features/networking/egress-ip.feature]: Yes, it is from that step.

File: features/networking/egress-ip.feature (hunk 3 above)
Comment (huiran0826) [reply features/networking/egress-ip.feature]: @anuragthehatter  Given as admin i successfuly merge step  will patch the egressnetworkpolicy to default project that is why I change it to this way that I can specify project.

Comment (anuragthehatter) [reply features/networking/egress-ip.feature]: Ah right, if your raw file doesn't support namespace key then not possible. /LGTM

**Comment by huiran0826** [comment]
@zhaozhanqi @weliang1 @anuragthehatter @rbbratta. 
Any other comments?

**Pull request merged**